keepassxc-browser/translate.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const items = document.querySelectorAll('[data-i18n]');
+for (let item of items) {
+    const key = item.getAttribute('data-i18n');
+    if (key) {
+        const placeholder = item.getAttribute('i18n-placeholder');
```

Comment:
Is this the standard name? If not, why is it not another `data-*` attribute? Perhaps you could also find a shorter name? `tr()` is intentionally short.

Reply:
There's no standard to this, AFAIK. The reason it's not another `data-*` attribute is that I wanted it to describe it's related to the translations. Not just a plain placeholder that could be for any data.